### ISSUE TITLE
Finger contractures

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9687]
**Preferred term label:**
Finger contractures

Shahrzad and Noemi, How should the existing terms under [Camptodactyly of finger HP:0100490](https://hpo.jax.org/app/browse/term/HP:0100490) be integrated with the proposed new finger contracture terms?

**Comment by shahrzadnema** [comment]
The suggestions are:
Level 1: (existing term) Joint contracture of the hand (HP:0009473)
Level 2: (new term) Finger contracture
Level 3: 
- (existing term) Flexion contracture of finger (HP:0012785) 
- AND 
- (new term) extension contracture of finger
Camptodactyly of finger (HP:0100490) is already linked to the Flexion contracture of the Proximal interphalangeal joint of the 2nd finger (HP:HP:0009540), 3rd (HP:0009471), 4th (HP:0009276), and 5th (HP:0009185), so no new suggestions from our end!

**Comment by pnrobinson** [comment]
added Finger joint contracture and extension contracture of finger as suggested
